src/tests.c

```diff
@@ -5208,6 +5208,7 @@ int main(int argc, char **argv) {
     }
     secp256k1_rand_seed(seed16);
 
+    setbuf(stdout, NULL);
```

Comment:
C89 says `setbuf` should be called before any operation is performed on the stream. This means that it is currently in the right position but I think it would be less fragile to put at the beginning of `main()`. (For example, someone could have the idea of changing the `fprintf(stderr, ...)`  in line 5195 to `stdout`.) Also, I think this call deserves a comment with a rationale.

Reply:
Good point. I moved the call to the beginning of main and added comments.